nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ][OPTIONAL] Well done implementing the requirement as pointed by the previous reviewer. However, this could have included a custom error message like so 👇🏾 

```ruby
class Nameable
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
end

```